src/main/java/nl/tudelft/contextproject/controller/GameController.java

```diff
@@ -246,4 +248,15 @@ public void gameEnded(boolean didElvesWin) {
 		Main main = Main.getInstance();
 		main.setController(new EndingController(main, didElvesWin));
 	}
+
+	/**
+	 * Method used for testing.
+	 * sets the hud to a new hud.
```

Comment:
"sets the hud to a new hud" can be removed, as that should be the case with a setter